metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -79,7 +79,9 @@ final class SyntheticsDecorationProvider(
     for {
       focused <- focusedDocument()
       filePath = AbsolutePath(path)
-      sourcePath <- SemanticdbClasspath.toScala(workspace, filePath)
+      sourcePath <-
+        if (!filePath.isScala) SemanticdbClasspath.toScala(workspace, filePath)
+        else Some(filePath)
```

Comment:
Out of curiosity, was this causing issues before? In reality, what other types of files would actually be hitting this? Java files?

Reply:
Ach, so this was normally only getting semanticdb files (which would correspond to some scala file), but now that we can produce them from standalone files, then that's the path that is being sent.